performance_manager/lib/gtfs_static_table.py

```diff
@@ -247,11 +247,17 @@ def process_static_tables(db_manager: DatabaseManager) -> None:
     """
     process gtfs static table files from metadataLog table
     """
+    process_logger = ProcessLogger("load_static_tables")
+    process_logger.log_start()
 
     # pull list of objects that need processing from metadata table
     paths_to_load = get_unprocessed_files("FEED_INFO", db_manager.get_session())
+    process_logger.add_metadata(filecount=len(paths_to_load))
 
     for folder, folder_data in paths_to_load.items():
+        individual_logger = ProcessLogger("load_static_table", folder=folder)
+        individual_logger.log_start()
+
```

Comment:
This for loop with technically be loading a whole set of static tables. So the `individual_logger` would he loading a whole set of static tables with the same timestamp. And then the `process_logger` would be loading multiple sets of static tables with different timestamps (this won't actually happen in normal operation)